app/lib/app/SecvisogramPage/View/WizardTab/editors/GenericEditor.js

```diff
@@ -46,17 +47,13 @@ export default function Editor({ parentProperty, property, instancePath }) {
   const { doc, collectIds } = React.useContext(DocumentEditorContext)
 
   const uiType = property.metaData?.uiType
-  const label = property.title || property.metaData?.title || 'missing title'
-
   const disabled =
     loginAvailable && userInfo
       ? property.metaData?.disable?.ifServerMode || false
       : property.metaData?.disable?.ifStandaloneMode || false
+  const label = t(`csaf.${property.metaData?.i18n?.title}`)
+  const description = t(`csaf.${property.metaData?.i18n?.description}`)
```

Comment:
The `property.metaData?.i18n?.title` could be `undefined`, same as in `ObjectEditor`